### PR TITLE
Task/sc 1101/node js 12 deprecation

### PR DESCRIPTION
**Pull request by Fabian6781**
## Jira Story
SC-1101

## About

1. actions/checkout@v2 -> actions/checkout@v3
2. actions/setup-python@v1 -> actions/setup-python@v4
3. aws-actions/configure-aws-credentials@v1 -> aws-actions/configure-aws-credentials@v1-node16

## Associated Pull Requests
N/A

## Update Changelog

* [X] Changelog updated

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (gordon-epc) [inline .github/workflows/ci.yml:1]: Looks like this one was missed:
`actions/upload-artifact@v2`

**Review by gordon-epc — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,3 +12,7 @@ For instructions, see the [changelog confluence page](https://epcpower.atlassian
 - SC-760: Pin poetry to 1.1.15 in CI
 - SC-572: Added changelog for release notes.
 
+### CI
+
+- SC-1101: Update ci.yml actions to fix poetry version issue
```
Comment (gordon-epc) [inline CHANGELOG.md]: Let's use the same changelog message as the original grid-tied fix.
```suggestion
- SC-1101: Pin versions of all python packages for installing poetry
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -33,26 +33,30 @@ jobs:
...
       - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v1
+        uses: aws-actions/configure-aws-credentials@v1-node16
         with:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ env.AWS_REGION }}
       - name: Install Poetry and Venv
         run: |
+          #install Poetry
```
Comment (gordon-epc) [inline .github/workflows/ci.yml:53]: Not necessary, as the `name` above says the same thing.
```suggestion
```

**Review by gordon-epc — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Fabian6781) [reply CHANGELOG.md]: Done.

**Review by gordon-epc — APPROVED** [review]

**Pull request merged**